src/app/core/components/left-menu/left-menu.component.html

```diff
@@ -12,25 +12,32 @@
       <div class="menu-header-desktop" [class.show-shadow]="showArchiveOptions">
         <div class="archive">
           <div class="archive-thumb" prBgImage [bgSrc]="archive.thumbURL500" [cover]="true"></div>
-          <div class="archive-name">The {{archive.fullName}} Archive</div>
-          <div class="archive-access">Access role: {{archive.accessRole | prConstants}}</div>
-          <div class="archive-options-toggle" (click)="showArchiveOptions = !showArchiveOptions"
+          <div class="archive-name-access">
+            <div class="archive-options-toggle" (click)="showArchiveOptions = !showArchiveOptions"
```

Comment:
This isn't new behavior, so feel free to tell me this is out of scope, but I find it frustrating for the archives options to shut after clicking on one - particularly when the menu collapse animation completes before the dialog box (or whatever) begins animating in. Can we make it stay open? Should that be part of this?

Reply:
This is a really good suggestion and I think that should be part of the ticket because it should be relatively simple. The ticket *is* called "Improvements to Left Side Menu", and this is an improvement!

Reply:
Awesome, this feels great!